src/apiController.js

```diff
@@ -0,0 +1,217 @@
+let axios = require('axios');
+let config = require('/src/config.js');
+
+
+axios.defaults.baseURL = config.server;
+axios.defaults.headers.common['Authorization'] = config.token;
+
+//I've basically created functions for each type of call. Didn't actually have them do anything yet because I haven't fully figured out how to use these.
+// They'll also need to have actual stuff written in them to actually use them, but I figure people can fill in the info for their own functions as we go on.
+let apiController = {
+
+
+  // Products API calls
+  getProduct: (pages, countNum) => {
```

Comment:
Being a little nitpicky probably, but I feel like this would be better named "getProducts", as I assume it's going to fetch an array of products rather than just one. 

Also, I think it would be better to have the parameters (for all of them, not just this one) be destructured from an object, so that we dont necessarily have to remember the order. You're already assigning default values for all of the parameters, so I think it literally boils down to just adding curly braces around your parameter lists, and functionality shouldn't change.

I'm also on my phone right now, so unfortunately I can't pull down the code and actually run it.